gossip/src/cluster_info.rs

```diff
@@ -553,6 +553,22 @@ impl ClusterInfo {
                 }
 
                 let node_version = self.get_node_version(node.pubkey());
+                
+                let (node_version_str, client_str) = node_version.as_ref().map_or(
```

Comment:
I would add new client in to the _version/src/lib.rs:119_ to avoid code duplication.